### DISCUSSION TITLE
Thresholds for individual request

### DISCUSSION THREAD

**Discussion by george4joseph** [apideck-libraries/postman-to-k6#51] [Q&A]
Is it possible in postman-to-k6 generation - thresholds for individual request?
eg
```
postman[Request]({
    name: "Request Copy",
    id: "adbe51ca-6e8c-420c-b2c8-b62e6f81ab71",
    method: "GET",
    address: "{{base_url}}/sample?",
    headers: {
      "Content-Type": "application/vnd.api+json",
      Accept: "application/vnd.api+json",
      Authorization: "{{token}}"
    },
    tags: { name: "Request Copy" }
  });
```
  For all request need to check thresholds.

**Comment by thim81** [comment]
@george4joseph 

I don't think that it is possible to define threshold per request.
This is more linked to K6 execution it self.

There is an option `Threshold on group duration` or `Thresholds on tags`
https://k6.io/docs/using-k6/thresholds/#threshold-on-group-duration
https://k6.io/docs/using-k6/thresholds/#thresholds-on-tags

Another strategy could be to generate multiple K6 scripts, where you organise your requests per threshold, but this would be part of structuring your requests.
